source/task-5.html.md.erb

```diff
@@ -13,177 +18,168 @@ These instructions assume that you have prototype changes ready to push to GitHu
 
-6. To use the token, click the __copy__ icon to copy the token to your clipboard. Save this somewhere safe; once you navigate away from the page you will no longer be able to view it.
-<br>
-<br>
-![Click generate new token](/images/copy-token.png)
+5. Give your token a name and select the **repo** checkbox.
 
-You can now use the token in the command line to perform git operations.
+    This gives you full access to private and public repositories from within the command line.
 
-```
-$ git clone https://github.com/username/repo.git
+6. Select the **Generate token** button once you're done.
+
+7. To use the token, select the **copy** icon to copy the token to your clipboard.
+
+    Save this somewhere safe. Once you navigate away from the page you will no longer be able to view it.
+
+You can now use the token in the terminal to perform git operations:
+
+```shell
+git clone https://github.com/username/repo.git
 Username: your_username
 Password: your_token
 ```
 
 ## Deploy with Heroku
 
-Open [www.heroku.com](https://www.heroku.com/) and log in if you're not already logged in.
+1. Go to [www.heroku.com](Heroku) and log in if you're not already logged in.
 
-In the top right click **New** then **Create new app**
+2. Select **New**, then **Create new app**.
 
-Names in Heroku have to be unique across all the users of Heroku. It can be helpful to add your name or organisation to the start of the name to make it unique. For example **yourname-juggling-prototype**
+3. Choose an **App name**.
 
-Select **Europe** for the region - this is not important but makes your prototype a bit faster.
+    Names in Heroku have to be unique across all the users of Heroku. It can be helpful to add your name or organisation to the start of the name to make it unique. For example 'yourname-juggling-prototype'.
 
-For **deployment method** choose **GitHub**
+4. Select **Europe** as the region - this is not important but makes your prototype a bit faster.
 
-![Deploy Heroku app using github master branch](/images/heroku-deploy.png)
+5. Select **Create app**.
  
-1. Scroll down and click **Connect to GitHub**
+6. Set **Deployment method** as **GitHub**, then select **Connect to GitHub**.
 
-2. In the popup, click **Authorize Heroku**
+7. In the popup, select **Authorize Heroku**.
 
-3. In the **repo-name** field, type all or some of your repo name, as it is on GitHub. Click **search**
+8. In the **repo-name** field, Type all or some of your repo name, as it is on GitHub. Select **search**.
 
-4. Click **connect** on the right of your repo.
+9. Select **connect** on the right of your repo.
 
-5. Scroll down to the **Manual deploy** section and click **Deploy branch**
+10. Scroll down to the **Manual deploy** section and select **Deploy branch**.
 
 ### Secure your prototype
 
-One more thing: we need to set a username and password or the prototype won't run online.
+One more thing: we need to set a username and password or the prototype will not run online.
```

Comment:
Should this be 'you need to set a username...'?

Reply:
Good spot, thank you